fe1-web/src/core/objects/Server.ts

```diff
@@ -0,0 +1,63 @@
+import { RemoveMethods } from 'core/types';
+
+import { PublicKey } from './PublicKey';
+
+export type ServerAddress = string;
+
+export interface ServerState {
+  address: string;
+  publicKey: string;
+}
+
+export class Server {
```

Comment:
1. The name is a problem. This class doesn't implement a server. I'd go with something clearer, such as `ServerInfo`, `ServerReference`, etc. 
2. Should this class be in `core/objects` ? Where do we really need this ?

Reply:
1. I agree 👍
2. This really boils down to what we want to have in the core and what not. Basically the core could also just be a feature that does not accept a config but only exports its interface. My thought was that it might be used by a other features as well and since it (for now) is not a message but a json rpc method that stores instances I put it in the core. If we had it as a feature and as a json rpc method at the same time, we would need to extend the interface to intercept other types of json rpc calls?